pavelib/assets.py

```diff
@@ -213,12 +312,105 @@ def compile_coffeescript(*files):
 @no_help
 @cmdopts([
     ('system=', 's', 'The system to compile sass for (defaults to all)'),
+    ('themes_dir=', '-td', 'The themes dir containing all themes (defaults to None)'),
+    ('themes=', '-t', 'The theme to compile sass for (defaults to None)'),
     ('debug', 'd', 'Debug mode'),
     ('force', '', 'Force full compilation'),
 ])
 def compile_sass(options):
     """
-    Compile Sass to CSS.
+    Compile Sass to CSS. following is a list of some possible ways to use this command.
+
+    Command:
+        1. paver compile_sass
+    Description:
+        compile sass files for both lms and cms
```

Comment:
Am I correct in that the themes are not compiled by default? IMO the default option should be to compile all the themes, because that will ensure that production/stage/sandbox environments will have the necessary CSS files generated. As it stands, it seems like there's an extra manual step to generate the SASS for any themes. Is it expected that there's a theme installation script that creates a centralized themes directory, copies the necessary directories, and then runs the SASS compilation?